### ISSUE TITLE
Reversed map axes for sswidl sunpos files

### ISSUE THREAD

**Issue by ianan** [NuSTAR/nustar_pysolar#36]
For a variety of *sunpos.evt files I'm looking at, which have been sunpos converted via sswidl, the axes of the maps in python are reversed. 

This is due to map.py using the x value of TCDLT from the fits header to define both the x and y pixel size (cdelt1 and cdelt2) and the SSWIDL conversion makes this value negative, i.e.([https://github.com/NuSTAR/nustar_solar/blob/master/util/nustar_convert_to_solar.pro#L113](https://github.com/NuSTAR/nustar_solar/blob/master/util/nustar_convert_to_solar.pro#L113))

So a quick solution could be just to use the absolute value in map.py?
`delx = abs(hdr['TCDLT'+xval])`

Like we do when making maps in sswidl.

**Comment by bwgref** [comment]
Yeah...I think in the python version I switch the X values directly while in the SSWIDL version I put this into the TCDLT value. We can go ahead and make the change here with a note that it's for backward compatibility to the SSWIDL version of the conversion code.

**Comment by bwgref** [comment]
...this must have only been used previously to make ds9 work out nicely, right? In SSWIDL we must use the X values directly rather than using the TCDLT values, right?

**Comment by ianan** [comment]
No, the TCDLT value is used in sswidl, but only the absolute value.
